libcst/metadata/scope_provider.py

```diff
@@ -406,16 +406,18 @@ class Scope(abc.ABC):
     #: Refers to the GlobalScope.
     globals: "GlobalScope"
     _assignments: MutableMapping[str, Set[BaseAssignment]]
-    _accesses: MutableMapping[str, Set[Access]]
     _assignment_count: int
+    __accesses_by_name: MutableMapping[str, Set[Access]]
+    __accesses_by_node: MutableMapping[cst.CSTNode, Set[Access]]
```

Comment:
why the double underscore?

Reply:
Because there were other examples (in Access and BaseAssignment) where we use double underscores. Fixed